README.md

```diff
@@ -36,6 +36,72 @@ Deploy and verify a contract
 source .env
 
 # To deploy and verify our contract
-forge script script/WikiNoValidator.s.sol:WikiNoValidator --rpc-url $RPC_URL  --private-key $PRIVATE_KEY --broadcast --verify --etherscan-api-key $ETHERSCAN_KEY -vvvv --gas-price 60 --legacy
+forge script script/BrainPassDeployer.s.sol:BrainPassDeployer --rpc-url $RPC_URL  --private-key $PRIVATE_KEY --broadcast --verify --etherscan-api-key $ETHERSCAN_KEY -vvvv --gas-price 60 --legacy
 
 ```
+
+# Expalaining How BrainPassCollectibles Works
+
+BrainPassCollectibles is a Solidity smart contract that enables the creation and management of BrainPass NFTs. It allows users to buy passes to have some Wiki previledges on IQ Wiki and provides functionalities to mint NFTs, increase pass time, and manage pass types.
+
+## Usage
+
+### Contract Deployment
+
+Deploy the `BrainPassCollectibles` contract by providing the address of the IQ token contract as a constructor parameter.
+
+### Adding a Pass Type
+This function allows the contract owner to add a new pass type.
+
+Call the `addPassType` function with the following parameters:
+- `pricePerDay`: The price per day in IQ tokens for the pass.
+- `tokenURI`: The base URI for the pass token metadata.
+- `name`: The name of the pass type.
+- `maxTokens`: The maximum number of tokens that can be minted for this pass type.
+- `discount`: The discount percentage applied to the pass price (optional).
+
+### Minting an NFT Pass
+This function allows users to mint a BrainPass NFT for a specific pass type and duration.
+Call the `mintNFT` function with the following parameters:
+- `passId`: The ID of the pass type to mint.
+- `startTimestamp`: The start timestamp for the pass.
+- `endTimestamp`: The end timestamp for the pass.
+- the duration is calculated by endTimestamp - startTimestamp
+- An address can only mint one passtype.
+- The payment is in IQ Token
+
+### Increasing Pass Time
+This function is ued to increase the duration for which a pass is owned for.
+Call the `increasePassTime` function with the following parameters:
+- `tokenId`: The ID of the NFT whose time should be increased.
+- `newStartTime`: The new start timestamp for the pass.
```

Comment:
why would you need to pass a new start time? it should be the current now() time

Reply:
Ive not updated the readme doc, it was from before